src/main/java/com/google/cloud/tools/managedcloudsdk/components/BundledPythonCopier.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.managedcloudsdk.components;
+
+import com.google.cloud.tools.managedcloudsdk.command.CommandExecutionException;
+import com.google.cloud.tools.managedcloudsdk.command.CommandExitException;
+import java.util.Map;
+
+public interface BundledPythonCopier {
+  /**
+   * Copy a cloud sdk bundled python into a temporary directory and return an environment map with
+   * the location of that bundled python.
```

Comment:
`map defining {@code CLOUDSDK_PYTHON} set to the location of that copied python`?

BTW, from the interface point of view, it feels a bit weird that the method "copyPython" returns a map. What do think of returning a path set let callers to do whatever they want to do?

Reply:
Yeah that's true, I can move this around.

Reply:
I tried moving it around, but I think as a shared piece of code, the result is a little more easily consumable like this.